content/en/agent/guide/gcp-private-service-connect.md

```diff
@@ -0,0 +1,126 @@
+
+{{< img src="agent/guide/psc/gcp-psc-overview-1.png" alt="GCP Private Service Connect schema. On the left, a 'Customer VPC' box contains Datadog Agents sending data to a PSC endpoint. On the right, a 'Datadog VPC' box contains a service attachment in communication with Datadog services. The PSC endpoint in the 'Customer VPC' box connects to the service attachment in the 'Datadog VPC' box through the GCP Private Backbone. " >}}
+
+## Setup
+
+### Connect an endpoint
+
+1. In your GCP console, navigate to **Network services** > **Private Service Connect**.
+2. Go to the **Endpoints** section. Click on **Connect endpoint**.
+   {{< img src="agent/guide/psc/connect-endpoint.png" alt="Screenshot of a 'Connect endpoint' page in the GCP console" >}}
+   
+   - Under **Target**, select _Published service_.
+   - For **Target service**, enter the _PSC target name_ that corresponds to the Datadog intake service that you want to use. You can find your PSC target name in the [table of published services](#published-services).
+   - For **Endpoint name**, enter a unique identifier to use for this endpoint. You can use `datadog-<SERVICE>`. For example: `datadog-metrics`.
+   - For **Network** and **Subnetwork**, choose the network and subnetwork where you want to publish your endpoint.
+   - For **IP address**, click the dropdown and select _Create IP address_ to create an internal IP from your subnet dedicated to the endpoint. Select this IP.
+   - Check **Enable global access** if you intend to connect the endpoint to virtual machines outside of the `us-central1` region.
+
+   **Note**: Datadog exposes PSC producer endpoints from the `us-central1` region. These endpoints support global access, allowing services to connect from any region. However, the forwarding rule must be created in the `us-central1` region.
+
+3. Click **Add endpoint**. Verify that your status is _Accepted_. Take note of the IP address, as this is used in the next section.
+   {{< img src="agent/guide/psc/connect-endpoint-success.png" alt="Screenshot of a success message after adding an endpoint in the GCP console. Includes an IP address" >}}
+
+### Create a DNS zone
+1. In your GCP console, navigate to **Network services** > **Cloud DNS**.
+2. Click on **Create zone**.
+   {{< img src="agent/guide/psc/create-a-dns-zone.png" alt="Screenshot of a 'Create a DNS zone' page in the GCP console" >}}
+
+   - Under **Zone type**, select _Private_.
+   - For **Zone name**, enter a descriptive name for your zone.
+   - For **DNS name**, enter the _private DNS name_ that corresponds to the Datadog intake service that you want to use. You can find your DNS name in the [table of published services](#published-services).
+3. Next, create an `A` record that points to the endpoint IP. On the _Zone details_ page of the zone you created, click on **Add record set**.
```

Comment:
Does the user need to know what an `A` record is?

Reply:
although it might help the user to conceptually understand what they're about to do (an `A` record maps a hostname to an IPv4 address), in this situation the user only really needs a functional definition: an `A` record is one where you select `A` from the dropdown under "resource record type". this direction is explicit in the instructions that follow.

also, `A` records should be fairly common knowledge in the devops world. of course, relying on a user's assumed knowledge is a little dangerous—but in this case where they don't *need* the definition to successfully complete this configuration, i think it's fine to omit it.